Governance/concepts/authorship.md

```diff
@@ -0,0 +1,51 @@
+# Authorship
+
+Research output are usually the result of collaboration, so it is important to attribute the right roles to anyone involved.
+Often researchers don't distinguish between different levels of involvements and either add only the main authors, failing to recognise minor but still important contributions to the work, or they add anyone involved as an author.
+However, `author` is not the only possible option, most data portals and publishing standards now allow to use `contributor` to cover a range of different roles.
+ 
+```{note}
+Depending on the standard 
+```
```

Comment:
I think we could rethink the use of {note}s, esp on this page where they kind of seem out of place in the formatted version. This one is more of a footnote, but appears as a highlighted section of importance.

Reply:
thanks I'll double checked, that nearly looks like something got lost!!

Reply:
My bad I started writing the note and never finished, it's completed now. I kind use notes also to break down too much text. I removed the second Notes that could have just been added as a sentence.